src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.logo-img {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  height: 60px;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+}
```

please follow the BEM structure and move :hover to the nav_link block
```suggestion
.nav__link {
  display: flex;
  position: relative;
  align-items: center;
  white-space: nowrap;
  text-decoration: none;
  text-transform: uppercase;
  font-size: 12px;
  font-weight: 500;
  color: #000;
  height: 60px;

  &:hover {
   color: var(--main-color);
 }
}
```